container_fixtures/rabbitmq/rabbitmq.conf

```diff
@@ -0,0 +1,6 @@
+ssl_options.cacertfile           = /etc/ssl/certs/ca_certificate.pem
+ssl_options.certfile             = /var/lib/rabbitmq/server_rabbit.trento.local_certificate.pem
+ssl_options.keyfile              = /var/lib/rabbitmq/server_rabbit.trento.local_key.pem
+ssl_options.verify               = verify_peer
+ssl_options.fail_if_no_peer_cert = true
+listeners.ssl.default = 5671
```

Comment:
nitpick: we could indent the value to be the same as the others :see_no_evil:

Reply:
done!